.simplecov

```diff
@@ -0,0 +1,4 @@
+SimpleCov.start do
+  SimpleCov.minimum_coverage 100.00
```

Comment:
nice

Reply:
would it be worth adding rubocop? how many violations would the current version of the gem have?

Reply:
I'd like to. Let's give it a shot and see!

Reply:
Done!